OPHD/States/MapViewStateTurn.cpp

```diff
@@ -710,7 +716,10 @@ void MapViewState::nextTurn()
 
 	updateResidentialCapacity();
 
-	if (mPopulation.getPopulations().size() > 0)
+	int turnsSinceLanding = mTurnCount - mTurnNumberOfLanding; // If negative, landing has not yet occurred.
+
+	// Colony will not have a crime rate until at least n turns from landing, depending on difficulty
+	if (turnsSinceLanding > gracePeriod[mDifficulty])
```

Comment:
Was just thinking this suggestion could have gone further, and set a `bool` variable, rather than repeating a boolean expression in each `if` condition.

```cpp
bool isMoraleEnabled = mTurnCount > mTurnNumberOfLanding + gracePeriod[mDifficulty];
```